onnx/backend/test/case/node/groupnorm.py

```diff
@@ -0,0 +1,64 @@
+# SPDX-License-Identifier: Apache-2.0
+
+import numpy as np
+
+import onnx
+
+from ..base import Base
+from . import expect
+
+
+class GroupNormalization(Base):
+    @staticmethod
+    def export() -> None:
+        def _groupnorm_test_mode(x, num_groups, scale, bias, epsilon=1e-5):
```

Comment:
Nit: Suggest making this a top-level function (move it outside the class), as a "reference implementation". Also, suggest dropping the suffix `_test_mode` (feels like a copy-paste artifact not relevant to this op, since it doesn't have a separate test-mode).